data/templates/cryingfloor/config.yml

```diff
@@ -0,0 +1,10 @@
+name: Crying on Floor
+link: https://i.imgflip.com/ysuv0.jpg
```

Comment:
Can you provide a reference URL here? The point is to explain the source of the meme. I don't recognize this one.